Makefile

```diff
@@ -71,21 +71,6 @@ apply:
 check-style: webapp/node_modules
 	@echo Checking for style guide compliance
 
-ifneq ($(HAS_WEBAPP),)
-	cd webapp && npm run lint
-	cd webapp && npm run check-types
-endif
-
-ifneq ($(HAS_SERVER),)
-	@if ! [ -x "$$(command -v golangci-lint)" ]; then \
-		echo "golangci-lint is not installed. Please see https://github.com/golangci/golangci-lint#install-golangci-lint for installation instructions."; \
-		exit 1; \
-	fi; \
-
-	@echo Running golangci-lint
-	golangci-lint run ./...
-endif
-
```

Comment:
Did you mean to remove this? It looks like `make check-style` doesn't do anything any more.

Also, I think this section is common for all plugin projects, so it might be worth keeping it that way to keep this closer to the other plugins

Reply:
Yup, we can keep this, reverting this back. 🙌